00-programming-fundamentals/github-and-the-command-line.md

```diff
@@ -0,0 +1,100 @@
+# GitHub and the Command Line
+
+So far, we've only used git locally to keep track of files on our computer. However, much of the power of git is in moving files from place to place, and enabling collaboration between people. So let's learn about how to do some of that!
+
+You might want to bookmark this page.
+
+## Learning Goals
+
+At the end of this lesson you should be able to...
+
+* Pull the contents of a GitHub repository to your computer
+* Push your changes back to GitHub
+
+## Cloning a GitHub Repository
+
+This is the procedure you'll follow at the beginning of almost every project you work on.
+
+1. Fork the repository under your own GitHub account
+1. Check the address bar and repo title, and verify you're now on your fork
+1. In the middle left, just above the list of files, click the green `Clone or Download` button
+1. Copy the URI that pops up (or click the `Copy to clipboard` button)
+1. Open a terminal, and navigate to your `projects` folder
+1. _Clone_ your fork of the repository to your computer using the `git clone` command:
+  ```
+  $ git clone <paste the URI here>
+  ```
+1. The `git clone` command should have created a directory with the same name as the repo. `cd` into it.
+1. Proceed to work on your project. Remember to commit regularly!
+
+You won't need to run `git init` in a cloned project - that's something you only do if you're starting from scratch.
+
+### Remotes
+
+In git, repositories on other computers are called _remotes_. When you ran `git clone`, part of the setup process involved adding a remote called `origin`, which points to your fork of the project on GitHub. This will be important later.
+
+You can see a list of all remotes using the `git remote -v` command (the `-v` is short for _verbose_):
+
+```
+$ git remote -v
+origin	git@github.com:droberts-ada/Solar-System.git (fetch)
+origin	git@github.com:droberts-ada/Solar-System.git (push)
+```
+
+You should see two identical entries, one for `fetch` and one for `push`. The URI should have your GitHub username and the repo name in it, and should **not** contain `Ada-CX`.
+
+## Pushing Local Changes to GitHub
+
```

Comment:
Maybe some visual reference to help understand what's happening at each stage of this.